.pre-commit-hooks.yaml

```diff
@@ -22,3 +22,12 @@
   pass_filenames: false
   files: ^(.*/)?poetry\.lock$
   args: ["-f", "requirements.txt", "-o", "requirements.txt"]
+
+- id: poetry-update
+  name: poetry-update
```

Comment:
IMO, this ID/name is a bit misleading since the hook does not run `poetry update`. Maybe, `poetry-sync` would be a better name?

Reply:
💯